docs/en/sql-reference/statements/insert-into.md

```diff
@@ -13,12 +13,59 @@ Basic query format:
 INSERT INTO [db.]table [(c1, c2, c3)] VALUES (v11, v12, v13), (v21, v22, v23), ...
 ```
 
-The query can specify a list of columns to insert `[(c1, c2, c3)]`. In this case, the rest of the columns are filled with:
+You can specify a list of columns to insert using  the `(c1, c2, c3)` or `COLUMNS(c1,c2,c3)` syntax. 
+Insert statements can have asterisk with column transformers in the column list.
+
+For example:
+
+``` sql
+SHOW CREATE [db.]table
```

Comment:
This example doesn't work. I mentioned in one of previous reviews. Both the rest examples too.